parts/windows/kuberneteswindowsfunctions.ps1

```diff
@@ -215,17 +215,6 @@ function Register-NodeResetScriptTask {
     Register-ScheduledTask -TaskName "k8s-restart-job" -InputObject $definition
 }
 
-function Register-NodeLabelSyncScriptTask {

```

Comment:
The script is included in the sign-script, considering we needs a new release of the script and there's no harm to keep it there, as the first step, we remove the task creation